### ISSUE TITLE
Add Loading Wheel to Individual Data Layers

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#414]
Waiting for feedback from stakeholders before starting this

**Comment by ndellicarpini** [comment]
do you mean in the data layer list? they're already there in the legend

**Comment by jamesdoyle21** [comment]
No, in the data layer tool itself, like OceansMap 1 has. We need more outside opinion on this, so hold off for now
